genlib path fixes

steveri opened the pull request:
Oops, the latest full-chip build failed with the following error:

https://buildkite.com/tapeout-aha/fullchip/builds/368
```
assert File( 'outputs/glb_top_tt.lib' )
E   AssertionError: assert File( 'outputs/glb_top_tt.lib' )
E    +  where File( 'outputs/glb_top_tt.lib' ) = File('outputs/glb_top_tt.lib')
```

This set of changes will fix that, I think.

steveri commented on mflowgen/full_chip/glb_top/get_glb_top_outputs.sh:
```diff
@@ -9,7 +9,7 @@ else
 fi
 
 mkdir -p outputs
-cp -L *cadence-genus-genlib/outputs/design.lib outputs/glb_top_tt.lib
```
Notice the mismatch between `cadence-genus-genlib` specified here vs. `cadence-innovus-genlib` in `glb_top/construct.py` below, which apparently was also genus up until about Feb. 25. I went for a generic fix by wildcarding the entire prefix so that it works for both genus and innovus, but it's fine with me if you want to go a different way...

steveri replied:
huh won't this just break again when/if we switch from innovus back to genus?

kongty commented:
@steveri I removed comments and hard-fixed script to copy lib files from `*innovus-genlib` as `genus` has a problem with generating lib file. I want to be safe by forcing it to copy from `innovus-genlib` node.

steveri commented:
> @steveri I removed comments and hard-fixed script to copy lib files from `*innovus-genlib` as `genus` has a problem with generating lib file. I want to be safe by forcing it to copy from `innovus-genlib` node. I also rebased it to `master` branch

Okay thanks Taeyoung.
